requirements/openvino.txt

```diff
@@ -1,7 +1,7 @@
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
```

Comment:
Hi @ashwinvaidya17,
A numpy conflict is occurring in the existing requirements (numpy 1.19) for OTX detection training
So I want to upgrade the versions of openvino and nncf..
Could you please confirm that there are no other issues with this change on Anomaly?
(This PR is not ready for review, so there is no need for a separate review at the moment. I will make a request later.)

Reply:
@harimkang Yeah it shouldn't be a problem. Please go ahead 👍